src/openforms/forms/tests/test_api_formdefinition.py

```diff
@@ -39,6 +39,29 @@ def test_list(self):
 
         self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
 
+    def test_list_pagination(self):
+        user = StaffUserFactory.create(user_permissions=["change_form"])
+        self.client.force_authenticate(user=user)
+
+        FormDefinitionFactory.create_batch(100)
+
+        url = reverse("api:formdefinition-list")
+
+        with self.subTest("no pagination"):
+            response = self.client.get(url, {"page_size": "0"}, format="json")
+            self.assertEqual(response.status_code, status.HTTP_200_OK)
+            self.assertEqual(len(response.json()["results"]), 100)
```

Comment:
It appears the JSON response data is still formatted as a paginated response. Could it return the list of form definitions directly instead?